### ISSUE TITLE
t2t-vit-t and tnt train issue

### ISSUE THREAD

**Issue by EstherJinyan** [open-mmlab/mmclassification#768]
Hello,I encountered some problems when training the model, detailed questions are as follows:
I extracted 5 categories from cifar10 as datasets, and trained resnet and t2t-vit-t and tnt respectively. I found that only the loss and accuracy of resnet are normal, please help to look at a few other problems.

resnet:According to (https://github.com/open-mmlab/mmclassification/blob/master/configs/resnet/resnet50_8xb32_in1k.py) modification. The loss is from 1 to 0.01, and the accuracy rate is from 95% to 98%, which looks normal.

Q1:
t2t-vit-t: According to (https://github.com/open-mmlab/mmclassification/blob/master/configs/t2t_vit/t2t-vit-t-19_8xb64_in1k.py) modification.The problem is that although the accuracy rate is increasing during the training process, the initial accuracy rate is too low, only 20%. It seems that the pre-training model has not been successfully loaded, and the accuracy rate is only 80% after 18 epochs. Is this normal?My configuration file is attached below.

Q2:
tnt:According to(https://github.com/open-mmlab/mmclassification/blob/master/configs/tnt/tnt-s-p16_16xb64_in1k.py)modification.The problem is that as the number of training increases, the accuracy does not increase but decreases, and the loss hardly decreases.What is the reason for this?
Epoch [1] loss: 0.7199 accuracy_top-1: 96.0800
Epoch [2] loss: 0.4659 accuracy_top-1: 96.4800
Epoch [3] loss: 0.4906 accuracy_top-1: 95.1600
Epoch [4] loss: 0.4927 accuracy_top-1: 94.0600
Epoch [5] loss: 0.4828 accuracy_top-1: 92.7400
...

[t2t-tnt_config.zip](https://github.com/open-mmlab/mmclassification/files/8440041/t2t-tnt_config.zip)

Thanks a lot!

**Comment by Ezra-Yu** [comment]
For Q1:
 the t2t use [EMAhook](https://github.com/open-mmlab/mmclassification/blob/5a67bc8a1aefd5493732b8f6ab10ee8c04839773/configs/t2t_vit/t2t-vit-t-19_8xb64_in1k.py#L34), you can delete this line and try again. For more detail about EMA, you can refer to [this paper](http://proceedings.mlr.press/v37/ioffe15.html).

For Q2：
your dataset is very small, the networks are always designed for the `ImagNet-1k dataset`, your trainning settings have caused [`overfitting`](https://www.v7labs.com/blog/overfitting). please try to use smaller base-lr, smaller architectures networks, and simpler data augments.

And the image size of the `cifar10` dataset is much smaller, you can try common image size dataset, such as [cat-dog](https://github.com/open-mmlab/mmclassification/blob/master/docs/en/tutorials/MMClassification_tools.ipynb) dataset and [`CUB`](https://github.com/open-mmlab/mmclassification/pull/703).

**Comment by EstherJinyan** [comment]
No problem,thank you very much!
